src/fidesops/schemas/connection_configuration/__init__.py

```diff
@@ -63,7 +68,12 @@ def get_connection_secrets_validator(
     validated against a PostgreSQL schema.
     """
     try:
-        return secrets_validators[connection_type]
+        schema = (
+            SaaSSchemaFactory(saas_config).get_saas_schema()
+            if saas_config
+            else secrets_validators[connection_type]
+        )
+        return schema
```

Comment:
If the saas config hasn't been set on the ConnectionConfig, you'll get a 500 error here, https://github.com/ethyca/fidesops/issues/182 prevents this